### ISSUE TITLE
Is there some way to get the softmax (probability values) from the saved models using the repos code.

### ISSUE THREAD

**Issue by saksham-s** [ikhlestov/vision_networks#29]
Hi, is there some way to get the softmax probability values from the saved models instead of the final class predictions.

**Comment by ikhlestov** [comment]
Yes, of course. You just need to modify code a little bit and fetch required values under the session scope. For example if you want to fetch logits you need:
modify those lines - https://github.com/ikhlestov/vision_networks/blob/master/models/dense_net.py#L343-L344 with code like: `self.logits = logits`
and later on provide some additional method:
```
def fetch_logits(self, inputs):
    feed_dict = {self.images: inputs}
    logits_res = self.sess.run([self.logits], feed_dict=feed_dict)
    return logits_res
```
With such approach actually you can get values for any tensor, from saved or newly initialized model
